keyboards/akb/raine/info.json

```diff
@@ -0,0 +1,379 @@
+{
+    "keyboard_name": "raine",
+    "maintainer": "e11i0t23",
+    "width": 16.25,
+    "height": 5.25,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {
```

Comment:
Your object ordering here doesn't match the ordering used by your layout macro or `keymap.c`, and there's also a key missing (72 here, 73 in the layout macro). This file needs to be rebuilt.

If you need help with this, let me know.